flatgfa-py/src/lib.rs

```diff
@@ -126,6 +127,97 @@ impl PyFlatGFA {
         mmap.flush()?;
         Ok(())
     }
+    
+    #[getter]
+    fn size(&self) -> usize {
+        let gfa = self.0.view();
+        file::size(&gfa)
+    }
+
+    fn print_gaf_lookup(&self, gaf: &str) {
+        let gfa = self.0.view();
+
+        let name_map = flatgfa::namemap::NameMap::build(&gfa);
+
+        let gaf_buf = flatgfa::memfile::map_file(&gaf);
+        let parser = flatgfa::ops::gaf::GAFParser::new(&gaf_buf);
+
+        // Print the actual sequences for each chunk in the GAF.
+        for read in parser {
+            print!("{}\t", read.name);
+            for event in flatgfa::ops::gaf::PathChunker::new(&gfa, &name_map, read) {
+                event.print_seq(&gfa);
+            }
+            println!();
+        }
+    }
+
+    fn test_gaf(&self, gaf: &str) -> Vec<Vec<PyChunkEvent>> {
```

Comment:
Seems like this could have a more descriptive name: like, `all_reads` or something.